website/content/docs/integrations/consul-connect.mdx

```diff
@@ -1,70 +1,70 @@
 ---
 layout: docs
-page_title: Consul Connect
+page_title: Consul Service Mesh
 description: >-
-  Learn how to use Nomad with Consul Connect to enable secure service to service
+  Learn how to use Nomad with Consul service mesh to enable secure service to service
   communication
 ---
 
-# Consul Connect
+# Consul Service Mesh
 
 ~> **Note:** This guide requires Nomad 0.10.0 or later and Consul 1.6.0 or
 later.
 
-~> **Note:** Nomad's Connect integration requires Linux network namespaces.
-Nomad Connect will not run on Windows or macOS.
+~> **Note:** Nomad's service mesh integration requires Linux network namespaces.
+Consul service mesh will not run on Windows or macOS.
 
-[Consul Connect](https://www.consul.io/docs/connect) provides
+[Consul service mesh](https://www.consul.io/docs/connect) provides
 service-to-service connection authorization and encryption using mutual
 Transport Layer Security (TLS). Applications can use sidecar proxies in a
 service mesh configuration to automatically establish TLS connections for
-inbound and outbound connections without being aware of Connect at all.
+inbound and outbound connections without being aware of service mesh at all.
 
-# Nomad with Consul Connect Integration
+# Nomad with Consul service mesh integration
```

Comment:
Shouldn't it still be Title Cased in headings as in `# Consul Service Mesh` above?

Reply:
Will make title case. In the Learn style guide, there is only one H1 per page (the proper title) and all other headings use  sentence case.  We might not be there on docs yet.
```suggestion
# Nomad with Consul Service Mesh Integration
```